cot/src/admin.rs

```diff
@@ -135,6 +136,32 @@ async fn authenticate(request: &mut Request, login_form: LoginForm) -> cot::Resu
     }
 }
 
+/// Struct representing the pagination of objects.
+#[derive(Debug, Clone, Copy)]
```

Comment:
I think it won't hurt to add these, too, and might be potentially useful given it's a public `struct`.

```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```